src/components/gui/gui.jsx

```diff
@@ -45,6 +59,11 @@ const GUIComponent = props => {
         tabSelected: classNames(tabStyles.reactTabsTabSelected, styles.isSelected)
     };
 
+    const onAddExtensionClick = () => {
```

Comment:
In general we would define this function in the container and pass it in, rather than define it in the component. Obviously it is very temporary so no need to fix it, but just FYI.

Reply:
I think I would lean to moving it in the container now for hygiene — so the container/component split remains clear.